raianalysis/requirements.txt

```diff
@@ -0,0 +1,7 @@
+numpy>=1.17.2
+pandas>=0.25.1
+scipy>=1.4.1
+scikit-learn>=0.22.1
+lightgbm>=3.1.1
+interpret-community
+fairlearn
```

Comment:
Why not pin all of them?

Reply:
pinning makes it more difficult for users to install a package in an existing environment or to integrate it with other packages

Reply:
Then we're just asking for people to complain why things don't work. Perhaps I expressed myself badly. I meant "why not put a min version"? Fairlearn pre-v0.5.0 shouldn't really be used because of massive breaking changes. @riedgar-ms probably has thoughts, too.

Reply:
I would request an inclusive lower bound of 0.5.0 on Fairlearn, though :-)

Reply:
oh ok I added lower bound pins